lights/blink1_light.go

```
@@ -0,0 +1,52 @@
+    l.current = state
+    return nil
+}
+
+func (l *Blink1Light) Clear() error {
+    l.isOn = false
+    l.current = StateOff
+    return nil
+}
+
+func (l *Blink1Light) Close() {
+    l.Clear()
```

golangci-lint (1.62.2)

Error return value of `l.Clear` is not checked

(errcheck)